canonical-web-and-design/ubuntu.com
# Fix col width for raspberry image

## Done

- Fix raspberry image position hero section on `/download/raspberry-pi`

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/raspberry-pi
- Check the raspberry image position is correct in hero section 


## Issue / Card

Fixes #8574 

## Screenshots

![image](https://user-images.githubusercontent.com/58959073/96990498-1b3bb780-151f-11eb-8336-a595e717468c.png)
